telemetry-schema.md

```diff
@@ -0,0 +1,71 @@
+## Attributes
+
+Key | Description
+--- | ---
+event | `perf-report` (can this take other values?)
```

Comment:
No however there are other telemetry requests about the state of the addon:

stateKey = "enabled", "disabled", "UIDisabled", "UIOk", "uninstalled", "loaded" - the states the extension can go through. 

I think there are also default ones that come from Shield too here.